app/models/registration_payment.rb

```diff
@@ -11,23 +11,27 @@ class RegistrationPayment < ApplicationRecord
 
   delegate :auto_accept_preference_live?, to: :registration
   after_create :auto_accept_hook, if: :auto_accept_preference_live?
```

Comment:
Shouldn't this also become a `after_save` then?

Reply:
Not until we want to add Live AA support for manual payments I don't think - which I'm hoping will be not too many PR's away